### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by FintanK**
**Add homepage URL to package.json file**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "name": "aframe",
   "version": "0.2.0",
   "description": "Building blocks for the VR Web",
+  "homepage" : "https://aframe.io/"
```
Comment (ngokevin) [inline package.json]: missing comma

Comment (FintanK) [inline package.json]: Thanks!

**Comment by dmarcos** [comment]
@FintanK Can you squash your commits?

**Comment by FintanK** [comment]
Sure!

**Pull request closed without merging**